api/server/server.go

```diff
@@ -752,7 +723,7 @@ func (s *Server) postImagesCreate(version version.Version, w http.ResponseWriter
 		if !output.Flushed() {
 			return err
 		}
-		sf := streamformatter.NewStreamFormatter(useJSON)
+		sf := streamformatter.NewStreamFormatter(true)
```

Comment:
also if nobody uses NewStreamFormatter with `false`, you can remove the param to the function and refactor that package